src/style/app.css

```diff
@@ -479,3 +479,15 @@ body {
 .step-three-question {
   padding: 10vh;
   color: #000000; }
+
+@media (min-width: 960px) {
+  body:nth-child(2) {
```

Comment:
very strange selector. Could we add comment for what we have it ?

Reply:
that is selector for 'root'. i've decided not to add className for the root-element and not to use selector '#root' in scss-file

Reply:
It is ALWAYS better to use separate selector to style something - it is better in performance and better in support / reading the code )

Reply:
@ivan-kolesen please add separate class to root